src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasBySample.java

```diff
@@ -65,7 +65,7 @@ public void annotate(final ReferenceContext ref,
         Utils.nonNull(gb);
 
         if ( likelihoods == null || !g.isCalled() ) {
-            logger.warn("Annotation will not be calculated, genotype is not called or alleleLikelihoodMap is null");
+            logger.warn(AnnotationUtils.generateMissingDataWarning(vc, g, likelihoods));
```

Comment:
So here we have a regular logger instead of a `OneShotLogger`. I think we should be consistent: either have a regular logger in both `StrandBiasBySample` and `DepthPerSampleHC`, or a `OneShotLogger` in both places. For debuggability, it might actually be better to keep them both as regular loggers for now.